src/components/Article.jsx

```diff
@@ -1 +1,31 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({ title, date, text }) => {
+  const dateISO = date.toISOString().slice(0, 10);
+  const dateLocaleString = date.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+
+  return (
+    <div className="container">
+      <h1 className="title is-2">React Article</h1>
```

Comment:
better to move text to the new line - it looks more clear this way
```suggestion
      <h1 className="title is-2">
        React Article
      </h1>
```